docs/getting-started.md

```diff
@@ -27,75 +27,336 @@ npm create @workleap/project@latest <output-directory>
 
-The shell itself is a very thin [API layer](/reference) on top of [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/) and [React Router](https://reactrouter.com) with the goal of maximizing both libraries forces and staying as most as possible out of their ways.
+The shell itself is a very thin [API layer](/references) on top of [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/) and [React Router](https://reactrouter.com) with the goal of maximizing both libraries forces and staying as most as possible out of their ways.
 
-### Why Webpack Module Federation?
+### Webpack Module Federation
 
 We identified **2 major problems** with frontend federated applications:
-1. How to prevent loading the same large dependencies twice when switching between *modules*?
-2. How to offer a cohesive experience that doesn't feels *modular*?
+- How to prevent loading the same large dependencies twice when switching between *modules*?
+- How to offer a cohesive experience that doesn't feels *modular*?
 
 We believe [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/) solves the first problem by offering a mecanism capable of **deduping at runtime** the **common dependencies shared** by the **host** application and the **remote** modules. 
 
 With this mecanism in place, all federated parts of an application can now be loaded in the same [browsing context](https://developer.mozilla.org/en-US/docs/Glossary/Browsing_context) instead of nested browsing contexts (like [iframes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe)). 
 
 By sharing the same browsing context (e.g. the same [Document](https://developer.mozilla.org/en-US/docs/Web/API/Document), the same [Window object](https://developer.mozilla.org/en-US/docs/Web/API/Window), and the same [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model)), federated parts are now unified and **form a single application**, which solves the second issues.
 
-With [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/), we think that we have the possibility of developing federated applications that feels like monolithic applications from a user perspective.
+With [Webpack Module Federation](https://webpack.js.org/concepts/module-federation/), we believe that we will be able to develop federated applications that feels like monolithic applications :rocket:
 
-### Why React Router?
+### React Router
 
-Well, what would you rather use? Joking aside, [React Router](https://reactrouter.com) version 6 nesting routing feature is ideal for federated application as it makes the UI heavily composable and decoupled.
+React Router [nested routes](https://reactrouter.com/en/main/start/tutorial#nested-routes) feature is ideal for federated application as it makes the UI heavily composable and decoupled. Anyway, which else would you use? :joy:
```

Comment:
https://tanstack.com/router/v1
https://github.com/molefrog/wouter

Reply:
Haha come on!

Would love to add support for TanStack router but it's so not ready.